src/routes.js

```diff
@@ -161,6 +161,10 @@ The following meta fields are supported for bottom-level routes:
 
   - standalone (default: false): If standalone is `true` then application layout
     elements like navigation bar, background color, etc are not rendered.
+
+  - skipAutoLogout (default: false): If `true`, no alert will be displayed when
```

Comment:
To make the default `false` rather than `undefined`, you'll need to update `normalizeMeta()` below. I like the idea of making it `true`/`false` like other route meta fields.